site/src/components/WorkspaceStatusBadge/WorkspaceStatusBadge.tsx

```diff
@@ -34,70 +21,72 @@ export const getStatus = (
   icon: React.ReactNode
 } => {
   const status = getWorkspaceStatus(build)
+  const { t } = i18next
```

Comment:
Here is another way we can access translations: directly from `i18next` (please note import above). I am accessing translations this way because I am working within a function, not a component (and you can't use hooks in functions).

Reply:
Why is the hook preferable when you have the choice?

Reply:
Oh is it because it takes the namespace?

Reply:
Definitely! But also, [ `useTranslation`](https://react.i18next.com/latest/usetranslation-hook) gets called when language changes (`i18next.changeLanguage`). Importing directly means translations are not updated when language changes. There are workarounds here, and we don't need to worry about it right now because we're not actually translating our app, but it's one thing less to worry down the road if we ever do translate.

Reply:
Great to know. Can you add something to Notion about this?

Reply:
Sure!